.github/workflows/test.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       uses: actions/setup-node@v1
```

Comment:
Haven't explicitly mentioned installing Java in CI as its by default installed in ubuntu.